sdk/search/azure-search-documents/src/test/java/com/azure/search/documents/test/environment/setup/AzureSearchResources.java

```diff
@@ -6,6 +6,7 @@
 import com.azure.core.test.utils.TestResourceNamer;
 import com.azure.core.util.Configuration;
 import com.azure.core.util.CoreUtils;
+import com.azure.core.util.DateTimeRfc1123;
```

Comment:
Is this import being used?

Reply:
It is not supposed to have any changes in this class. I will try to remove the diff here.